tests/providers/google/cloud/operators/test_dataflow.py

```diff
@@ -149,7 +149,7 @@ def test_exec(self, gcs_hook, dataflow_hook_mock, beam_hook_mock, mock_callback_
             "job_name": job_name,
             "region": TEST_LOCATION,
             'output': 'gs://test/output',
-            'labels': {'foo': 'bar', 'airflow-version': 'v2-1-0-dev0'},
+            'labels': {'foo': 'bar', 'airflow-version': 'v2-1-0'},
```

Comment:
Whilst already changing this could you make it use `airlow.__version__` or similar so we don't have to change this line again when we bump version

Reply:
Yeah. I will.